CHANGELOG.md

```diff
@@ -1,14 +1,19 @@
 # UNRELEASED
 
```

Comment:
Maybe we can add a commented line here that reads something like 
```
Add new entries that have not yet been released in this section
```
So it's clearer to those who contribute, and are not on our team. What do you think?